app/controllers/tweets_controller.rb

```diff
@@ -5,7 +5,7 @@ class TweetsController < ApplicationController
   PER_PAGE = 10
   require "date"
   def index
-    @q = Tweet.where(user_id: @user.id).ransack(params[:q])
+    @q = user_tweets.ransack(params[:q])
```

Comment:
メソッド化するのも良いですが、リレーションを使って以下のようにするのはどうでしょうか？
```suggestion
    @q = current_user.tweets.ransack(params[:q])
```

Reply:
メソッド一つ消えるのはありがたいです。取り入れます。